src/components/interfaces/MOBILE_API.xml

```diff
@@ -2014,6 +2014,13 @@
                 <param name="speed" type="Float" minvalue="0" maxvalue="500" mandatory="true" since="2.0" until="5.0"/>
```

Comment:
@ShobhitAd  I beleave that API version should be incremented within the release 5.2.
In case if it will be increased now, functionality will be unusable on current develop. Because version of API is 5.1